routes/comment.server.routes.js

```diff
@@ -0,0 +1,10 @@
+const comment = require('../controllers/comment.server.controller');
+
+/**
+ * Handles HTTP requests for the Comment module using Express.js route()
+ * @param app Express.js application object
+ */
+module.exports = function(app) {
+    app.route('/api/v1/comments')
+        .post(comment.commentCreate);
+}
```

Comment:
```suggestion
```
There is already a route for adding a comment to an existing post in routes/forum.server.routes.js